README.md

```diff
@@ -67,6 +67,9 @@ func ProxyHandler(ctx *fasthttp.RequestCtx) {
 
 	switch string(ctx.Path()) {
 	case "/echo":
+		// [OPTIONAL]: you can override path from `WithURL_OptionWS`
+		//             by providing "Override-Path" header  
+		// ctx.Request.Header.Set("Override-Path", "/real_echo")
```

Comment:
Provide an API to use a fixed builtin header (such as PROXY-OVERRIDE-PATH ?) rather than a literal string

Reply:
Now developer is free to use default header `proxy.DefaultOverrideHeader` or any other he wants to.